translations/it.yaml

```diff
@@ -0,0 +1,347 @@
+
+      building:
+        title: "Edificio {id}"
+        titleNew: Registra un nuovo edificio
+
+      entrance:
+        title: "Entrata {modelDescription}"
+        titleNew: Registra una nuova entrata
+
+      dwelling:
+        title: "Abitazione {id}"
+        titleNew: Registra nuova abitazione
+
+    linkedModels:
+      linkedEntries: "{modelName} collegati"
+      noLinkedEntries: "Non sono ancora collegati {modelName}"
+      linkEntry: "Collega {modelName}"
+      linkExistingBuilding: Collega edificio esistente (trasformazione / demolizione)
+      createEntry: |-
+        Registra {modelName}
+        {model, select,
+          building {(nuova costruzione)}
+          dwelling {(nuova costruzione)}
+          entrance {(nuova costruzione)}
+          other {}
+        }
+      or: oppure
+      removeLink: elimina collegamento
+      buildingInfo: |-
+        Rivolgersi al portale Housing-Stat per aggiungere un edificio esistente non ancora inserito nel REA.
+
+    linkBuildingModal:
+      linkBuilding: Collega edificio
+      cancel: Annulla
+
+    header:
+      status:
+        label: Stato
+        placeholder: TODO
+      statusSelector:
+        actionInfo:
+          Le modifiche dello stato rappresentano un passaggio naturale all'interno del ciclo di vita di un oggetto. Le correzioni dello stato offrono l'opportunità di correggere condizioni dello stato errate.
+        statusInfo: "Nuovi stati per modifiche dello stato ammessi:"
+        noNextStates: Nessuno (stato finale)
+        statusLabel: Nuovo stato
+        submitChangeLabel: Modifica stato
+        submitCorrectionLabel: Correggi stato
+        correctionDropTitle: Informazioni relative allo stato necessarie per una correzione
+        changeDropTitle: Informazioni relative allo stato necessarie per una modifica
+        confirmCorrection: Conferma correzione dello stato
+        confirmChange: Conferma modifica dello stato
+        cancel: Annulla
+
+      issues: "Pendenze in sospeso per {modelName} {modelId}"
+      noIssues: "Nessuna pendenza in sospeso per {modelName} {modelId}"
+      linkIssues: "Vai all'elenco completo delle pendenze per il progetto di costruzione {projectId}"
+
+    linkStreet:
+      missingLocalityHint: Inserisci dapprima il nome della località.
+      change: Modifica via
+      link: Collega via
+      heading: Collega via
+      search: Cerca via...
+      linkSuccess: Via collegata con successo.
+      linkError: Errore nel tentativo di collegare la via.
+      streetHint: Per registrare un'entrata occorre collegare una via.
+      searchHint: 'Nella casella di ricerca è possibile inserire un asterisco (*) come segnaposto per cercare parti del nome della via.'
+      searchHintExamples: Esempi
+      searchHintExamples1: 'Unter* : Unterfeldweg, Unterschlossweg'
+      searchHintExamples2: '*weg : Waldweg, Unterschlossweg'
+      searchHintExamples3: 'B*strasse : Bernstrasse, Bachstrasse'
```

Comment:
I think here we used locale specific examples

Reply:
That makes sense. I'd prefer to fix this in a future MR, so that the customer can make some fitting suggestion. Ok?